modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/TextStyles.kt

```diff
@@ -90,14 +102,16 @@ fun TextH40(
     modifier: Modifier = Modifier,
     textAlign: TextAlign? = null,
     color: Color = MaterialTheme.theme.colors.primaryText01,
-    maxLines: Int = Int.MAX_VALUE
+    maxLines: Int = Int.MAX_VALUE,
+    fontWeight: FontWeight = FontWeight.Medium,
+    disableScale: Boolean = false
```

Comment:
As the category pillar title and duration are limited to small widths, I'm using non-scaled font size for both otherwise they spread into multiple lines on a large display screen.

Reply:
That makes sense. 

I am still seeing some issues related to that though. In particular, "Technology" still gets wrapped (going to be hard to avoid that since there maybe even longer categories). Also, the time values still appear to be scaled when I increase my font size.

<img width="350px" src="https://user-images.githubusercontent.com/4656348/201943970-ee876c83-6ca6-487b-a698-bb1f99da4f20.png" />

Reply:
I fixed font size for the duration and increased text widths a bit as we have a little space available between pillars. This is a tricky problem to solve I bet. 

<img width=320 src="https://user-images.githubusercontent.com/1405144/202190456-ced49b52-86a8-42f0-aec0-a9717b890ab8.png"/>